org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/tests/devicestate.json

```diff
@@ -0,0 +1,34 @@
+{   "data" :
+    {
```

Comment:
I renamed this to devicedata since it now contains both the initial device state as well as the test specific data.

---
In reply to: [82066747](https://github.com/openT2T/translators/pull/106#discussion_r82066747) [](ancestors = 82066747)